sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jms/ServiceBusJmsContainerConfiguration.java

```diff
@@ -49,6 +60,7 @@ public AbstractServiceBusJmsAutoConfiguration(AzureServiceBusJmsProperties azure
         DefaultJmsListenerContainerFactory jmsListenerContainerFactory = new DefaultJmsListenerContainerFactory();
         configurer.configure(jmsListenerContainerFactory, connectionFactory);
         jmsListenerContainerFactory.setPubSubDomain(Boolean.TRUE);
+        jmsListenerContainerFactory.setErrorHandler(new DefaultErrorHandler());
```

Comment:
Do we need this?

Reply:
Better to keep the error stack trace then. I will get rid of it.